dandischema/models.py

```diff
@@ -381,7 +394,7 @@ class Organization(Contributor):
     identifier: Optional[RORID] = Field(
         None,
         title="A ror.org identifier",
-        description="Use an ror.org identifier for institutions",
+        description="Use an ror.org identifier for institutions.",
```

Comment:
FWIW, I generally dislike unnecessary trailing punctuation - just waste of bytes IMHO. If desired, makes it also easy to just add `.` consistently "in code"